src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
+import validator from 'validator';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+export function checkIfIncludes(movieProperty: string, query: string) {
+  return movieProperty.toLowerCase().includes(query.toLowerCase());
+}
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const validate = (input: string) => {
+    setQuery(validator.trim(input));
```

Comment:
i'm not sure but will 'space' work in our search panel? Demo link is not working so can't check. If 'space' doesn't work than better move trim method to function 'checkIfIncludes'.